lib/Service/CreditNoteService.php

```diff
@@ -98,11 +98,11 @@ public function preview($params = null, $opts = null)
      *
      * @throws \Stripe\Exception\ApiErrorException if the request fails
      *
-     * @return \Stripe\CreditNote
+     * @return \Stripe\Collection
      */
     public function previewLines($params = null, $opts = null)
     {
-        return $this->request('get', '/v1/credit_notes/preview/lines', $params, $opts);
+        return $this->requestCollection('get', '/v1/credit_notes/preview/lines', $params, $opts);
```

Comment:
Is this expected?

Reply:
Yes. This will fix a bug in pagination on these list methods.